packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-with-replication.ts

```diff
@@ -0,0 +1,68 @@
+import { PipelineProject } from '@aws-cdk/aws-codebuild';
+import { Pipeline, Artifact } from '@aws-cdk/aws-codepipeline';
+import { Key } from '@aws-cdk/aws-kms';
+import { Bucket } from '@aws-cdk/aws-s3';
+import { App, Stack, RemovalPolicy } from '@aws-cdk/core';
+import { ENABLE_CROSS_REGION_REFERENCES } from '@aws-cdk/cx-api';
+import { IntegTest } from '@aws-cdk/integ-tests';
+import { S3SourceAction, CodeBuildAction } from '../lib';
+
+
+const account = process.env.CDK_INTEG_ACCOUNT || process.env.CDK_DEFAULT_ACCOUNT;
+const app = new App({
+  treeMetadata: false,
+});
+app.node.setContext(ENABLE_CROSS_REGION_REFERENCES, true);
+const stack1 = new Stack(app, 'integ-pipeline-producer-stack', {
+  env: {
+    region: 'us-east-1',
+    account,
+  },
+});
+const stack2 = new Stack(app, 'integ-pipeline-consumer-stack', {
+  env: {
+    region: 'us-east-2',
+    account,
+  },
+});
+
+
+const key = new Key(stack1, 'ReplicationKey');
+const bucket = new Bucket(stack1, 'ReplicationBucket', {
+  encryptionKey: key,
+  autoDeleteObjects: true,
+  removalPolicy: RemovalPolicy.DESTROY,
+});
+
+const artifact = new Artifact();
+const pipeline = new Pipeline(stack2, 'Pipeline', {
+  crossRegionReplicationBuckets: {
```

Comment:
Created this integration test based off of a common cross region scenario

https://github.com/aws/aws-cdk/blob/1f03e8c40a682a3b5aae90560c84017cfe62762e/packages/@aws-cdk/aws-codepipeline/README.md#L320-L374